hwdb.d/60-keyboard.hwdb

```diff
@@ -2223,6 +2223,11 @@ evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:
 evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE*:pvr*:*
  KEYBOARD_KEY_76=f21                                    # Fn+F1 toggle touchpad
 
+# VAIO VJFH52
+evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFH52F11X*:pvr*:*
+ KEYBOARD_KEY_f7=f23		                        # Fn+F1 toggle touchpad off
+ KEYBOARD_KEY_f8=f22		                        # Fn+F1 toggle touchpad on
```

Comment:
The comment does not match with the report in #35644. Maybe one of the comment should be Fn+F1 -> Fn+F2 ?

Reply:
Sorry, it was a typo in the report. It has already been corrected https://github.com/systemd/systemd/issues/35644#issue-2742490832